src/pages/index.js

```diff
@@ -24,57 +29,59 @@ const initialCards = [
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg ",
   },
 ];
-
+console.log(initialCards);
 /* -------------------------------------------------------------------------- */
 /*                           //------ IMPORTED CODE                           */
 /* -------------------------------------------------------------------------- */
 
 import {
-  // openModal,
-  // closeModal,
-  // closeModalByEscape,
-  // profileEditModal,
-  // cardOpenModal,
-  // cardOpenModalCloseButton,
   cardsList,
-  // profileModalCloseButton,
   profileModalEditButton,
   profileName,
   profileDescription,
-  // inputProfileName,------------------------
-  // inputProfileDescription,----------------
-  // modalProfileSaveButton,
-  // modalProfileForm,
-  // cardAddModal,
   cardModalAddButton,
-  // cardAddModalCloseButton,
-  // cardAddModalSaveButton,
-  // cardAddModalForm,
-  // inputCardName,
-  // inputImageLink,
-  // modalBackgrounds,
+  // modalConfirmDeleteCard,
+  // profileAvatarEditMOdal,
+  profileAvatar,
+  profileAvatarOverlay,
 } from "../utils/constants.js";
 
-// import { enableClosingModalFeature } from "../utils/utils.js";
-
 import Card from "../components/Card.js";
 
 import { FormValidator } from "../components/FormValidator.js";
 
 import "./index.css";
 import PopupWithForm from "../components/PopupWithForm.js";
-// import Popup from "./components/Popup.js";
+
 import UserInfo from "../components/UserInfo.js";
 import PopupWithImage from "../components/PopupWithImage.js";
 import Section from "../components/Section.js";
+import Api from "../components/Api.js";
+import PopupWithConfirm from "../components/PopUpWithConfirm.js";
 
 /* -------------------------------------------------------------------------- */
 /*                                  test area                                 */
 /* -------------------------------------------------------------------------- */
+api
+  .getCurrentUser("/users/me")
+  .then((result) => {
+    console.log(result);
+    console.log("umm now what??");
+    profileName.textContent = result.name;
```

Comment:
Needs correction: please use UserInfo class instance methods for updating the info

Reply:
Done